src/styles/_mixins.scss

```diff
@@ -23,7 +23,6 @@
     margin: variables.$gap;
     color: colors.$black;
     border: none;
-    width: 100%;
```

Comment:
This is a little dangerous to change since it potenitially changes a lot of UI. Have somone looked through it all?

Reply:
I have solved the one place I found but there might be more. But I think it is more logical now